common/src/main/java/org/keycloak/common/util/KeystoreUtil.java

```diff
@@ -38,8 +39,8 @@
 public class KeystoreUtil {
 
     public enum KeystoreFormat {
-        JKS("jks"),
-        PKCS12("p12", "pfx"),
+        JKS("jks", "truststore"),
+        PKCS12("p12", "pfx", "pkcs12"),
         BCFKS("bcfks");
```

Comment:
It'd be great to mention the mapping of extensions to specific KS formats somewhere in the documentation. 

I see it's only in [Upgrading guide for KC 26.0.0](https://www.keycloak.org/docs/latest/upgrading/index.html#keystore-and-trust-store-default-format-change):

_If the file extension is .p12, .pkcs12 or .pfx, the format is PKCS12. If the file extension is .jks, .keystore or .truststore, the format is JKS. If the file extension is .pem, .crt or .key, the format is PEM._

IMO it makes sense to mention it. @shawkins WDYT? Could you please potentially handle it in a follow-up?

Reply:
Sure, this can be moved to the regular docs. It could also mention .ca is accepted as a pem trust file.

Reply:
Added